src/util/generatePreview.js

```diff
@@ -177,18 +175,15 @@ class PreviewGenerator {
   _attachCssLibrary(css) {
     const linkTag = this.previewDocument.createElement('link');
     linkTag.rel = 'stylesheet';
-
-    const base64encoded = base64.fromByteArray(textEncoder.encode(css));
-    linkTag.href = `data:text/css;charset=utf-8;base64,${base64encoded}`;
-    this._previewHead.appendChild(linkTag);
+    linkTag.innerHTML = css;
+    this._previewHead.insertBefore(linkTag, this._previewHead.firstChild);
   }
 
   _attachJavascriptLibrary(javascript) {
     const scriptTag = this.previewDocument.createElement('script');
-    const base64encoded = base64.fromByteArray(textEncoder.encode(javascript));
-    scriptTag.src =
-      `data:text/javascript;charset=utf-8;base64,${base64encoded}`;
-    this.previewBody.appendChild(scriptTag);
+    const javascriptText = String(javascript);
+    scriptTag.innerHTML = javascriptText.replace(/<\/script>/g, '<\\/script>');
```

Comment:
Will this kind of escaping maintain existing functionality, though? That’s a JS escape rather than an HTML escape so I fear not.

E.g., as a reduction, would this “library” continue to function?

```js
console.log('</script>');
```